### PR TITLE
Update and rename .env.example to .env

### PR DESCRIPTION
**Pull request by yassbek**

### CODE REVIEW

**Review by krumware — CHANGES_REQUESTED** [review]
.env is reserved and should not be a default file
the user needs to opt in to the license by adding the value manually

File: .env [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # - The codebase has to stay open source, whether it was modified or not
 # - You can not repackage or sell the codebase
 # - Acquire a commercial license to remove these terms by emailing: license@cal.com
-NEXT_PUBLIC_LICENSE_CONSENT=
+NEXT_PUBLIC_LICENSE_CONSENT='agree'
```
Comment (PeerRich) [inline .env:8]: correct. we keep this empty, however we will get rid of this .env variable once we have the proper onboarding wizard where you accept the license @zomars

Comment (zomars) [reply .env:8]: I don't think we can modify the env variable on runtime (wizard) but we can come up with another way to accept it maybe.

Comment (krumware) [reply .env:8]: Yea we can't really lose the .env altogether for runtime config, but there are probably other ways to handle the consent part specifically

Comment (PeerRich) [reply .env:8]: @zomars we dont need the license check in the .env at all. its just a reminder how the license works. it just needs to be a check in the onboarding wizard and saved in the database

**Pull request closed without merging**